localisation/00_zuwc_united_localisation.csv

```diff
@@ -164,7 +163,6 @@ EVTDESC_WCBDP_95_case_01;Our new allies have asked us to give them an opportunit
 EVTOPTA_WCBDP_95;Well, the Swamp of Sorrows - not the best place, but no one claims it.;;Nun, die S�mpfe des Elends sind vielleicht nicht der sch�nste Fleck Azeroths, aber niemand erhebt bisher Anspruch auf sie.;;;;;;;;;;;x
 EVTOPTA_WCBDP_95_tooltip;Your new allies build their settlement in the Swamp of Sorrows, while their leader has joined the expedition.\n;;Eure neuen Verb�ndeten errichten ihre Siedlungen in den S�mpfen des Elends, w�hrend ihr Anf�hrer sich eurer Expedition anschlie�t.\n;;;;;;;;;;;x
 EVTOPTB_WCBDP_95;What more will they eventually want?!;;Was wollen sie denn noch?!;;;;;;;;;;;x
-EVTOPTA_WCBDP_95_tooltip;You leave your new allies in the dying Draenor.;;Ihr lasst Eure neuen Verb�ndeten in der sterbenden Welt von Draenor zur�ck.;;;;;;;;;;;x
```

Comment:
Isn't this the tooltip to `EVTOPTB_WCBDP_95;What more will they eventually want?!;;Was wollen sie denn noch?!;;;;;;;;;;;x
` ?